implementations/nestjs-code-first/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "nest-typescript-starter",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Nest TypeScript starter repository",
+  "license": "MIT",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main"
+  },
+  "dependencies": {
+    "@apollo/subgraph": "^2.2.2",
+    "@nestjs/apollo": "^10.1.7",
+    "@nestjs/common": "^9.2.1",
+    "@nestjs/core": "^9.2.1",
+    "@nestjs/graphql": "^10.1.7",
+    "@nestjs/platform-express": "^9.2.1",
+    "apollo-server-express": "^3.10.0",
```

Comment:
should we use AS4 instead?

Reply:
i.e. https://www.npmjs.com/package/@as-integrations/next

Reply:
NestJS hasn't been updated to use AS4 yet unfortunately. That work is being tracked here: https://github.com/nestjs/graphql/issues/2445